### DISCUSSION TITLE
Split pane customization.

### DISCUSSION THREAD

**Discussion by ankitkk** [Eugeny/tabby#3709] [Q&A] [answered]
Right now the split panes kind a flow into each other. I would like to to setup a high contrast thin border around split panes. I believe it can be done with custom css.  Any help would be appreciated. Debugging tool leads me to .xterm css but not sure what to do next. zero css skills, here.

**Comment by Eugeny** [comment, marked as answer]
Yep - try this CSS snippet: 

```css 
split-tab-spanner {
  background: black;
}
```

**Comment by lawweiliang** [comment]
@ankitkk   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
